image/docker_schema2.go

```diff
@@ -166,13 +167,40 @@ func (m *manifestSchema2) UpdatedImage(options types.ManifestUpdateOptions) (typ
 	case "": // No conversion, OK
 	case manifest.DockerV2Schema1SignedMediaType, manifest.DockerV2Schema1MediaType:
 		return copy.convertToManifestSchema1(options.InformationOnly.Destination)
+	case imgspecv1.MediaTypeImageManifest:
+		return copy.convertToOCIManifest()
 	default:
 		return nil, errors.Errorf("Conversion of image manifest from %s to %s is not implemented", manifest.DockerV2Schema2MediaType, options.ManifestMIMEType)
 	}
 
 	return memoryImageFromManifest(&copy), nil
 }
 
+func (m *manifestSchema2) convertToOCIManifest() (types.Image, error) {
```

Comment:
yup, have it on my todo, just wanted to bring this PR up for general consensus on the way I'm fixing this.